docs/split_stats.md

```diff
@@ -0,0 +1,34 @@
+# Get Splits Stats 
+
+`get_splits(playerid, year=None, player_info=False, pitching_splits=False)`
+
+Look up a player's split stats from baseball-reference. Either batting or pitching splits can be returned and splits can be provided for any season or for career. 
+Split stats are returned as a multi-index dataframe by split category and split. 
+Additionaly, player info can also be returned as a dictionary since the information is available without requiring a separate request to the baseball-reference page. 
+
+## Arguments
+`playerid:` String. The player's bbref playerid. Example: Mike Trout is 'troutmi01'
+
+`year:` String. Optional. The year to get split stats for. Leaving this out will provide career split stats.
```

Comment:
This should probably be an int, not a string.

Reply:
Yea, an int will work and is probably the preferred type. I updated this to say Integer